DDL for table in non default database can't be read
![image](https://user-images.githubusercontent.com/82543000/219023992-4294f3c4-3e80-4791-8d46-9bc4cb187a23.png)
```
com.simba.spark.support.exceptions.GeneralException: [Simba][SparkJDBCDriver](500051) ERROR processing query/statement. Error Code: 0, SQL state: org.apache.hive.service.cli.HiveSQLException: Error running query: org.apache.spark.sql.AnalysisException: Table or permanent view not found: nyctaxi.trips; line 1 pos 18
	at org.apache.spark.sql.hive.thriftserver.HiveThriftServerErrors$.runningQueryError(HiveThriftServerErrors.scala:48)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation.$anonfun$execute$1(SparkExecuteStatementOperation.scala:498)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at com.databricks.unity.EmptyHandle$.runWith(UCSHandle.scala:124)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation.org$apache$spark$sql$hive$thriftserver$SparkExecuteStatementOperation$$execute(SparkExecuteStatementOperation.scala:410)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation$$anon$2$$anon$3.$anonfun$run$2(SparkExecuteStatementOperation.scala:321)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at org.apache.spark.sql.hive.thriftserver.ThriftLocalProperties.withLocalProperties(ThriftLocalProperties.scala:149)
	at org.apache.spark.sql.hive.thriftserver.ThriftLocalProperties.withLocalProperties$(ThriftLocalProperties.scala:49)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation.withLocalProperties(SparkExecuteStatementOperation.scala:54)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation$$anon$2$$anon$3.run(SparkExecuteStatementOperation.scala:299)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation$$anon$2$$anon$3.run(SparkExecuteStatementOperation.scala:284)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:422)
	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1878)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation$$anon$2.run(SparkExecuteStatementOperation.scala:333)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
Caused by: org.apache.spark.sql.AnalysisException: Table or permanent view not found: nyctaxi.trips; line 1 pos 18
	at org.apache.spark.sql.AnalysisException.copy(AnalysisException.scala:128)
	at org.apache.spark.sql.hive.thriftserver.SparkExecuteStatementOperation.$anonfun$execute$1(SparkExecuteStatementOperation.scala:491)
	... 19 more
, Query: SHOW CREATE TABLE nyctaxi.trips.

	at com.simba.spark.hivecommon.api.HS2Client.pollForOperationCompletion(Unknown Source)

	at com.simba.spark.hivecommon.api.HS2Client.executeStatementInternal(Unknown Source)

	at com.simba.spark.hivecommon.api.HS2Client.executeStatement(Unknown Source)

	at com.simba.spark.hivecommon.dataengine.HiveJDBCNativeQueryExecutor.executeNonRowCountQueryHelper(Unknown Source)

	at com.simba.spark.hivecommon.dataengine.HiveJDBCNativeQueryExecutor.executeQuery(Unknown Source)

	at com.simba.spark.hivecommon.dataengine.HiveJDBCNativeQueryExecutor.<init>(Unknown Source)

	at com.simba.spark.hivecommon.dataengine.HiveJDBCDataEngine.prepare(Unknown Source)

	at com.simba.spark.jdbc.common.SPreparedStatement.<init>(Unknown Source)

	at com.simba.spark.jdbc.jdbc41.S41PreparedStatement.<init>(Unknown Source)

	at com.simba.spark.jdbc.jdbc42.S42PreparedStatement.<init>(Unknown Source)

	at com.simba.spark.hivecommon.jdbc42.Hive42PreparedStatement.<init>(Unknown Source)

	at com.simba.spark.spark.jdbc.SparkJDBCObjectFactory.createPreparedStatement(Unknown Source)

	at com.simba.spark.jdbc.common.JDBCObjectFactory.newPreparedStatement(Unknown Source)

	at com.simba.spark.jdbc.common.SConnection$5.create(Unknown Source)

	at com.simba.spark.jdbc.common.SConnection$5.create(Unknown Source)

	at com.simba.spark.jdbc.common.SConnection$StatementCreator.create(Unknown Source)

	at com.simba.spark.jdbc.common.SConnection.prepareStatement(Unknown Source)

	at com.simba.spark.jdbc.common.SConnection.prepareStatement(Unknown Source)

	at com.simba.spark.jdbc.common.SConnection.prepareStatement(Unknown Source)

	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCConnectionImpl.prepareStatement(JDBCConnectionImpl.java:244)

	at org.jkiss.dbeaver.ext.databricks.model.DatabricksMetaModel.getTableDDL(DatabricksMetaModel.java:171)

	at org.jkiss.dbeaver.ext.generic.model.GenericTable.getObjectDefinitionText(GenericTable.java:70)

	at org.jkiss.dbeaver.model.struct.DBStructUtils.getTableDDL(DBStructUtils.java:119)

	at org.jkiss.dbeaver.model.struct.DBStructUtils.generateTableListDDL(DBStructUtils.java:137)

	at org.jkiss.dbeaver.model.sql.generator.SQLGeneratorDDL.run(SQLGeneratorDDL.java:65)

	at org.jkiss.dbeaver.ui.UIUtils.lambda$17(UIUtils.java:1753)

	at org.eclipse.jface.operation.ModalContext.runInCurrentThread(ModalContext.java:434)

	at org.eclipse.jface.operation.ModalContext.run(ModalContext.java:352)

	at org.eclipse.ui.internal.WorkbenchWindow.lambda$7(WorkbenchWindow.java:2346)

	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:74)

	at org.eclipse.ui.internal.WorkbenchWindow.run(WorkbenchWindow.java:2344)

	at org.eclipse.ui.internal.progress.ProgressManager$RunnableWithStatus.run(ProgressManager.java:1132)

	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:74)

	at org.eclipse.ui.internal.progress.ProgressManager.lambda$27(ProgressManager.java:1011)

	at org.eclipse.swt.widgets.Synchronizer.syncExec(Synchronizer.java:183)

	at org.eclipse.ui.internal.UISynchronizer.syncExec(UISynchronizer.java:133)

	at org.eclipse.swt.widgets.Display.syncExec(Display.java:4785)

	at org.eclipse.ui.internal.progress.ProgressManager.runInUI(ProgressManager.java:1011)

	at org.jkiss.dbeaver.ui.UIUtils.runInUI(UIUtils.java:1752)

	at org.jkiss.dbeaver.ui.UIUtils.runInUI(UIUtils.java:1765)

	at org.jkiss.dbeaver.ui.editors.sql.generator.SQLGeneratorDialog.createDialogArea(SQLGeneratorDialog.java:102)

	at org.jkiss.dbeaver.ui.editors.sql.generator.SQLGeneratorDialog.createDialogArea(SQLGeneratorDialog.java:1)

	at org.eclipse.jface.dialogs.Dialog.createContents(Dialog.java:767)

	at org.jkiss.dbeaver.ui.dialogs.BaseDialog.createContents(BaseDialog.java:74)

	at org.jkiss.dbeaver.ui.editors.sql.dialogs.ViewSQLDialog.createContents(ViewSQLDialog.java:84)

	at org.eclipse.jface.window.Window.create(Window.java:431)

	at org.eclipse.jface.dialogs.Dialog.create(Dialog.java:1094)

	at org.jkiss.dbeaver.ui.dialogs.BaseDialog.create(BaseDialog.java:89)

	at org.eclipse.jface.window.Window.open(Window.java:788)

	at org.jkiss.dbeaver.ui.editors.sql.generator.SQLGeneratorContributor$2.run(SQLGeneratorContributor.java:180)

	at org.eclipse.jface.action.Action.runWithEvent(Action.java:474)

	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:580)

	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:414)

	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)

	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4256)

	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1066)

	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4054)

	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3642)

	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)

	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)

	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)

	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)

	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:643)

	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)

	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:550)

	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)

	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:296)

	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)

	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)

	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)

	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)

	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)

	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)

	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)

	at java.base/java.lang.reflect.Method.invoke(Unknown Source)

	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)

	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)

```

--- a/plugins/org.jkiss.dbeaver.ext.databricks/src/org/jkiss/dbeaver/ext/databricks/model/DatabricksTable.java
+++ b/plugins/org.jkiss.dbeaver.ext.databricks/src/org/jkiss/dbeaver/ext/databricks/model/DatabricksTable.java
@@ -158,6 +158,7 @@ public class DatabricksTable extends GenericTable {
     public String getFullyQualifiedName(DBPEvaluationContext context) {
         return DBUtils.getFullQualifiedName(
             getDataSource(),
+            getCatalog(),
             getSchema(),
             this);
     }

--- a/plugins/org.jkiss.dbeaver.ext.databricks/src/org/jkiss/dbeaver/ext/databricks/model/DatabricksView.java
+++ b/plugins/org.jkiss.dbeaver.ext.databricks/src/org/jkiss/dbeaver/ext/databricks/model/DatabricksView.java
@@ -139,6 +139,7 @@ public class DatabricksView extends GenericView {
     public String getFullyQualifiedName(DBPEvaluationContext context) {
         return DBUtils.getFullQualifiedName(
             getDataSource(),
+            getCatalog(),
             getSchema(),
             this);
     }

